pkg/cmd/extension/init/cmd_test.go

```diff
@@ -100,58 +102,50 @@ func TestGetEnvoyExtensionInitValidateFlag(t *testing.T) {
 	}
 }
 
+// TODO: test default to current directory
+
 func TestGetEnvoyExtensionInit(t *testing.T) {
 	const extensionName = "getenvoy_extension_init"
 
 	type testCase struct {
 		name string
 		extension.Category
 		extension.Language
-		currentDirectory bool
 	}
 
 	var tests []testCase
 	for _, c := range extension.Categories {
 		for _, l := range extension.Languages {
 			name := fmt.Sprintf(`category=%s language=%s`, c, l)
-			tests = append(tests,
-				testCase{name + "-currentDirectory", c, l, true},
-				testCase{name + "-newDirectory", c, l, false},
-			)
+			tests = append(tests, testCase{name, c, l})
 		}
 	}
 
 	for _, test := range tests {
 		test := test // pin! see https://github.com/kyoh86/scopelint for why
 
 		t.Run(test.name, func(t *testing.T) {
-			outputDir, revertOutputDir := RequireNewTempDir(t)
-			defer revertOutputDir()
+			extensionDir, revertExtensionDir := RequireNewTempDir(t)
```

Comment:
`revert` make me think that this would revert something e.g. going back to the some directory, etc. So maybe it's more appropriate to use `tearDownDir` or similar because it's just deleting the tempdir, right?

Reply:
same goes for others

Reply:
will go with removeExtensionDir as it is less jargon than what I had before